types/constants.go

```diff
@@ -77,7 +77,7 @@ func init() {
 		// environments than a humand can interact with.
 		BlockFrequency = 1 // As fast as possible
 		MaturityDelay = 3
-		GenesisTimestamp = CurrentTimestamp()
+		GenesisTimestamp = CurrentTimestamp() - 1e6
```

Comment:
where are you seeing the race condition? I'm not seeing anything in the race detector, nor do I think that this causes any type of datarace.